src/lib/components/common/Checkbox.svelte

```diff
@@ -0,0 +1,111 @@
+<script lang="ts">
+  import { createEventDispatcher } from "svelte";
+  import { Check16, Dash16 } from "svelte-octicons";
+
+  const dispatch = createEventDispatcher();
+
+  export let disabled = false;
+  export let status: "off" | "on" | "some" = "off";
+  export let label = "";
+
+  let checkbox: HTMLInputElement;
+
+  $: checked = status === "on" || status === "some";
```

Comment:
Possible to do something like this?

```
export let indeterminate = false;
$: {
  checkbox?.indeterminate = indeterminate`
}
```

Reply:
Even easier: `<input type=checkbox bind:indeterminate>` and we're done.